src/notation/internal/notationnoteinput.cpp

```diff
@@ -313,7 +313,7 @@ void NotationNoteInput::endNoteInput(bool resetState)
     }
 
     if (resetState) {
-        is.setTrack(muse::nidx);
+        is.setTrack(0);
```

Comment:
I'm not too convinced about this one, in the constructors the value is set to`muse::nidx`, and this one here is not related to any system item, or is it?